samples/drivers/led/led_strip/boards/esp32s3_devkitc_procpu.overlay

```diff
@@ -20,10 +20,10 @@ i2s_led: &i2s0 {
 	dmas = <&dma 3>;
 	dma-names = "tx";
 
-	led_strip: ws2812 {
+	led_strip: ws2812@0 {
 		compatible = "worldsemi,ws2812-i2s";
 
-		i2s-dev = <&i2s_led>;
+		reg = <0>
```

Comment:
missing semicolon `;` here and at other places as well.

Reply:
Please make it a -1 as this should indeed be blocking
Kind of unfortunate this doesn't seem covered in CI